src/plenoptic/simulate/models/portilla_simoncelli.py

```diff
@@ -84,20 +84,19 @@ def __init__(
             tight_frame=False,
         )
         self.filterPyr = SteerablePyramidFreq(
-            self.pyr._lomasks[-1].shape[-2:], height=0, order=1,
+            getattr(self.pyr, f'_lomasks_scale_{n_scales-1}').shape[-2:],
```

Comment:
In general, yes it's better not to directly access hidden attributes. In this case, this line whole bit gets removed in #225 , so I'm going to keep this as is here. The other reason I hesitate to do that is because I don't want `lomasks` to be a public attribute of the steerable pyramid class -- users shouldn't be interacting with them.